cypress/integration/app_admin/app_admin.js

```diff
@@ -0,0 +1,49 @@
+import { Given, When, Then, Before, After } from 'cypress-cucumber-preprocessor/steps';
+
+const USER_NAME = 'test@unit.no';
+const APP_ADMIN_ROLE = 'App-admin';
+
+Before(() => {
+  cy.deleteUser(USER_NAME).then(() => {
+    cy.addUser(USER_NAME, APP_ADMIN_ROLE).then((userId) => {
+      cy.wrap(userId).as('userId');
+      cy.visit('/');
+      cy.wait(5000);
+
+      cy.get('[data-testid=create-author-button]').click();
+      cy.get('[data-testid=modal_next]').click();
+      cy.get('[data-testid=skip-connect-to-orcid]').click();
+      cy.get('[data-testid=menu]').click();
+
+      // Set language
+      cy.get('[data-testid=menu-user-profile-button]').click();
+      cy.get('[data-testid=language-selector]').click();
+      cy.get('[data-testid=user-language-eng]').click();
+      cy.visit('/');
+    });
+  });
+});
+
+Given('that the user is logged in', () => {
+  cy.get('@userId');
+});
+Given('they have the role of Application administrator', () => {});

```

Comment:
Hvorfor er det to Givens? Går det ikke an å ha `And`? Eller er det slik man skal gjøre det hvis man har en `And` etter `Given`?

Reply:
Kan ikke bruke `And` desverre. Er ikke støtte for det i Cypress så man må bruke flere `Given` (samme for `When` og `Then`)

Reply:
Der tok jeg feil, ser ut som de har lagt til støtte for And siden sist jeg jobbet med det. Byttet ut Given med And der det skal være.